src/Chapter1.hs

```diff
@@ -477,8 +477,8 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = mod (abs(n)) 10
```

Comment:
Nice one! 
Note, that you can remove `()` around the single variable `n`, they are redundant in this case. And in Haskell, arguments are usually separated by space from the function name and each other :slightly_smiling_face: 
```suggestion
lastDigit n = mod (abs n) 10
```